### ISSUE TITLE
 Sonoff 4CH и  Sonoff Dual

### ISSUE THREAD

**Issue by konstantin999** [tretyakovsa/Sonoff_WiFi_switch#68]
Добрый день.
Спасибо за отличную прошивку! 
Интересуюсь, будет ли работать прошивка на  Sonoff 4CH и Sonoff Dual?

**Comment by renat2985** [comment]
Будет, но управлять пока сможете только одним реле. В скором будущем добавим возможность управления до 8 реле.

**Comment by konstantin999** [comment]
Круто! Буду ждать. Есть шансы ускорить эту разработку донатом?

**Comment by Igor122017** [comment]
Здравствуйте Ренат. у меня проблема, подскажите как лучше поступить, у меня 2 выключателя sonoff wifi-rf? на одну зону, и на 2 зоны. на 1 зону я прошил вашей программой и все работает хорошо, а вот на 2 зоны, я прошил, но там оказался только 1 выключатель... в итоге, я остался без света. помогите или подскажите, как лучше поступить, чтобы заработал выключатель на 2 зоны. заранее спасибо

**Comment by Igor122017** [comment]
PS, выключатель sonof на 2 зоны, стоит чип 8285

**Comment by tretyakovsa** [comment]
@Igor122017 
Ссылку дайте на устройство.

**Comment by Igor122017** [comment]
может лучше фото платы? будет более понятно

**Comment by Igor122017** [comment]
https://www.itead.cc/smart-home/sonoff-t1.html  вот этот выключатель, только у меня на 2 зоны...

**Comment by Igor122017** [comment]
Добрый вечер Ренат, у вас есть прошивка для выключателя soniff T1 на 2 зоны?

**Comment by tretyakovsa** [comment]
@Igor122017 
Нет у нас такой поддержки  sonoff T1. Могу написать тестовый скетч вы его загрузите в свой модуль и  протестируете. Пришлете мне данные и я добавлю в нашу прошивку.

**Comment by Igor122017** [comment]
хорошо,  я готов протестировать и предоставлю вам нужные данные

**Comment by Igor122017** [comment]
и еще , может поможет, на сколько я понял: 1 выключатель сидит на GPIO5 и второй на GPIO9

**Comment by Igor122017** [comment]
да, и заранее спасибо за помощь... когда примерно можно будет сделать тест? вы сможете здесь ссылку сбросить, чтобы не искать нужную прошивку....
